pkgs/by-name/op/openscap/package.nix

```diff
@@ -0,0 +1,155 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  libsepol,
+  popt,
+  libxml2,
+  libxslt,
+  openssl,
+  util-linux,
+  pcre2,
+  pcre,
+  libselinux,
+  graphviz,
+  glib,
+  python3,
+  swig,
+  python3Packages,
+  libgcrypt,
+  opendbx,
+  xmlbird,
+  haskellPackages,
+  libyaml,
+  xmlsec,
+  bzip2,
+  valgrind,
+  asciidoc,
+  installShellFiles,
+  rpm,
+  system-sendmail,
+  hyperscan,
+  gnome2,
+  curl,
+  procps,
+  systemd,
+  perl,
+  doxygen,
+  pkg-config,
+  perl538Packages,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openscap";
+  version = "1.3.10";
+
+  src = fetchFromGitHub {
+    owner = "OpenSCAP";
+    repo = "openscap";
+    rev = version;
+    hash = "sha256-P7k+Ygz/XmpTSKBEqbyJsd1bIDVJ1HA/RJdrEtjYD5g=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cmake
+    asciidoc
+    doxygen
+    rpm
+    swig
+    util-linux
+    pkg-config
+  ];
+
+  buildInputs =
+    with perl538Packages;
+    [
+      XMLXPath
+      LinuxACL
+      XMLTokeParser
+    ]
+    ++ [
+      perl
+      popt
+      openssl
+      valgrind
+      pcre2
+      pcre
+      libxslt
+      xmlsec
+      hyperscan
+      libselinux
+      libyaml
+      xmlbird
+      installShellFiles
+      bzip2
+      python3
+      libgcrypt
+      libxml2
+      systemd
+      haskellPackages.pthread
+      graphviz
+      system-sendmail
+      procps
+      asciidoc
+      libsepol
+      curl
+      glib
+      gnome2.ORBit2
+      gnome2.GConf
+      opendbx
+    ];
+
+  prePatch = ''
+    substituteInPlace swig/perl/CMakeLists.txt --replace-fail "DESTINATION ''${PERL_VENDORLIB}" "DESTINATION $out/lib''${PERL_VERSION}"
+    substituteInPlace swig/perl/CMakeLists.txt --replace-fail "DESTINATION ''${PERL_VENDORARCH}" "DESTINATION $out/lib"
+    substituteInPlace src/common/oscap_pcre.c --replace-fail "#include <pcre2.h>" "#include <${pcre2.dev}/include/pcre2.h>"
+    substituteInPlace src/common/oscap_pcre.c --replace-fail "#include <pcre.h>" "#include <${pcre.dev}/include/pcre.h>"
+  '';
+
+  cmakeFlags = [
+    "-DPCRE2_INCLUDE_DIRS=${pcre2.dev}/include"
+    "-DPCRE2_LIBRARIES=${pcre2.out}/lib"
+    "-DENABLE_DOCS=TRUE"
+    "-DENABLE_TESTS=TRUE"
+    "-DENABLE_OSCAP_UTIL=TRUE"
+    "-DENABLE_OSCAP_UTIL_CHROOT=TRUE"
+    "-DENABLE_OSCAP_UTIL_SSH=TRUE"
+    "-DENABLE_OSCAP_UTIL_DOCKER=TRUE"
+    "-DENABLE_VALGRIND=TRUE"
+    "-DPYTHON_SITE_PACKAGES_INSTALL_DIR=lib"
+    "-DOPENSCAP_INSTALL_DESTINATION=bin"
+    "-DCMAKE_INSTALL_BINDIR=bin"
+    "-DCMAKE_INSTALL_MANDIR=share"
+    "-DENABLE_MITRE=TRUE"
+    "-DCMAKE_INSTALL_LIBDIR=lib"
+    "-DCMAKE_INSTALL_INCLUDEDIR=include"
+    "-DCMAKE_INSTALL_DATADIR=share"
+    "-DBUILD_TESTING=ON"
+    "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON"
+    "-DCMAKE_POLICY_DEFAULT_CMP0025=NEW"
+  ];
+
+  buildPhase = ''
+    make
+    make docs
+  '';
+
+  installPhase = ''
+    make install PREFIX=$out
```

Comment:
Aren't makeFlags working here?

Reply:
it is working. also consider this dependency for merge #327164

Reply:
merged :)

Reply:
openscap ready for merge now